skrub/_reporting/tests/test_table_report.py

```diff
@@ -123,6 +127,42 @@ def test_duration(df_module):
     assert re.search(r"2(\.0)?\s+days", TableReport(df).html())
 
 
+@pytest.mark.parametrize("filename_type", ["str", "Path", "file_object"])
+def test_write_html(pd_module, filename_type):
+    df = pd_module.make_dataframe({"a": [1, 2], "b": [3, 4]})
+    report = TableReport(df)
+
+    with TemporaryDirectory() as td:
+        tmp_file_path = Path(td) / Path("report.html")
+
+        if filename_type == "str":
+            filename = str(tmp_file_path)
+        elif filename_type == "file_object":
+            filename = open(tmp_file_path, "w", encoding="utf-8")
+        else:
+            filename = tmp_file_path
+
+        report.write_html(filename)
+        assert tmp_file_path.exists()
```

Comment:
we don't need to check the full content but maybe we could see that the file contains `"</html>`" to see that the full report has been written